src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -0,0 +1,59 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline asset` commands:
+    * snapshot
+    * upload
+    * diff
+    * download
+"""
+
+import click
+
+from .._common import _handle_error
+
+
+@click.group(name="asset")
+@_handle_error
+def cli_asset():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_asset.command(name="snapshot")
+@click.option("--root-dir", help="The root directory to snapshot.")
+@_handle_error
+def asset_snapshot(**args):
+    """
+    Creates manifest of files specified root directory.
+    """
+    click.echo("snapshot taken")
+
+
+@cli_asset.command(name="upload")
+@click.option("--manifest", help="The manifest to ")
+@_handle_error
+def asset_upload(**args):
+    """
+    Preupload files before for a render with specified manifest.
```

Comment:
This comment is explaining an example use case of the command rather than what it does. Can we change this to explain what the command does instead? e.g. something like `Uploads the assets in the provided manifest file to S3`)